packages/babel-types/src/builders/generated/deprecated.js

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
Why don't generate this as `.ts`?

Also, having `deprecated.d.ts` in `src` generates a `deprecated.d.js` file in `lib`.

Reply:
names of exports here are conflicting with names of AST nodes, and so to export only ast nodes in type definitions deprecated builders (starting with upper case, were placed in js file, to not include then in typechecks)